sematic/tests/test_log_reader.py

```diff
@@ -665,6 +674,141 @@ def test_load_log_lines(mock_storage, test_db, log_preparation_function):  # noq
     assert result.lines == ["Line 42"]
 
 
+@pytest.mark.parametrize(
+    "log_preparation_function",
+    (prepare_logs_v2,),
+)
+def test_load_log_lines_reverse(
+    mock_storage, test_db, log_preparation_function  # noqa: F811
+):  # noqa: F811
+    run = make_run(future_state=FutureState.CREATED)
+    save_run(run)
+    resolution = make_resolution(status=ResolutionStatus.SCHEDULED)
+    resolution.root_id = run.root_id
+    save_resolution(resolution)
+    max_lines = 50
+
+    result = load_log_lines(
+        run_id=run.id,
+        forward_cursor_token=None,
+        reverse_cursor_token=None,
+        max_lines=max_lines,
+        reverse=True,
+    )
+    result.forward_cursor_token = None
+    result.reverse_cursor_token = None
```

Comment:
Are these reset in preparation for the next assert?
Shouldn't they be checked to be non-None before resetting them?

Reply:
It is in preparation for the next assert. I can check whether they are non-None here, but I think it would be wasteful to do that every time. The main fields of `LogLineResult` to be interested in for most of these tests are `lines` and `log_info_message`.